dev-manuals/test-plans/minigames/bugfinder-frontend/plans/u-bugfinder-4-show-general-feedback.md

```diff
@@ -0,0 +1,44 @@
+# Bugfinder` Show general feedback (`u.bugfinder-4`)
+
+
+Version: V1.1, 05.08.2022, Improve format and wording	
+Author: Timo Schnaible, Leon Hofmeister
+Tester: -
```

Comment:
```suggestion
Version: V1.1, 05.08.2022, Improve format and wording

Author: Timo Schnaible, Leon Hofmeister

Tester: -
```
When all is on one line, it makes it harder to read, especially when there are comments about the updated version

Reply:
Not intended to be one line.
Will add explicit line breaks.